src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -163,7 +164,8 @@ public GhprbTrigger(String adminlist,
     }
 
     @Override
-    public Object readResolve() {
+    public Object readResolve() throws ObjectStreamException {
+        super.readResolve();
```

Comment:
So this is wrong...

as super.readResolve() may return a different instance than `this` and the subsequent checks only work if you got `this`